eox_core/settings/test.py

```diff
@@ -38,7 +38,11 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
 plugin_settings(SETTINGS)
 vars().update(SETTINGS.__dict__)
 
-INSTALLED_APPS += ('eox_core', )
+try:
+    import edx_when  # pylint: disable=unused-import
+    INSTALLED_APPS += ('eox_core', 'edx_when.apps.EdxWhenConfig')
```

Comment:
Why is this needed?

Reply:
@morenol edx-proctoring(2.4.0) adds this requirement and there is a runtime error if this is not included in the installed apps